packages/webpack/default-config.js

```diff
@@ -0,0 +1,144 @@
+  if (developmentEs5) {
+    return { es5: es5Target };
+  }
+
+  // development by default has es6 output
+  return { es6: es6Target };
+}
+
+// eslint-disable-next-line
+module.exports = userOptions => {
+  const options = {
+    ...defaultOptions,
+    ...userOptions,
+  };
+
+  const config = {
+    entry: Array.isArray(options.indexJS) ? options.indexJS : [options.indexJS],
+
+    output: {
+      filename: '[name].[chunkhash].js',
+      chunkFilename: '[name].[chunkhash].js',
+      path: path.resolve(process.cwd(), 'dist'),
+    },
+
+    resolve: {
+      mainFields: ['es2015', 'module', 'main'],
+      // // ensure all modules are resolved from the root node_modules folder
+      modules: ['node_modules'],
+    },
+
+    module: {
+      rules: [
+        // Transforms import.meta.url to original source file location
+        {
+          test: /\.js$/,
+          use: [
+            BabelMultiTargetPlugin.loader(),
+            path.resolve(__dirname, './loaders/import-meta-url-loader.js'),
+          ],
+        },
+      ],
+    },
+
+    plugins: [
+      // Babel configuration for multiple output bundles targeting different sets of browsers
+      // for development, we only have a modern browser output
+      new BabelMultiTargetPlugin({
+        babel: {
+          plugins: [
+            // Minify HTML and CSS in tagged template literals
+            [
+              BabelPluginHTMLMinifier,
+              {
+                modules: { 'lit-html': ['html'], 'lit-element': ['html', 'css'] },
+                htmlMinifier: { collapseWhitespace: true, minifyCSS: true, removeComments: true },
+              },
+            ],
+            // fast async uses Promise instead of regenerator to compile async functions
+            'module:fast-async',
+            // use modules for babel helpers to avoid duplication, disable regenerator we use fast-async
+            ['@babel/plugin-transform-runtime', { regenerator: false, useESModules: true }],
+          ],
+
+          // @babel/preset-env options common for all bundles
+          presetOptions: {
+            useBuiltIns: 'usage',
+            // we use fast-async, it's faster than regenerator
+            exclude: ['transform-async-to-generator', 'transform-regenerator'],
+          },
+        },
+        safari10NoModuleFix: true,
+        targets: getBrowserTargets(options),
+      }),
+      new HtmlWebpackPlugin({
+        template: options.indexHTML,
+        inject: true,
+      }),
+    ],
+
+    devtool: ENV === 'development' ? 'inline-source-map' : 'source-map',
+    devServer: {
+      contentBase: process.cwd(),
+      compress: true,
+      port: 8080,
+      historyApiFallback: true,
+      stats: {
+        stats: 'errors-only',
+      },
+    },
+  };
+
+  if (ENV === 'production') {
+    config.plugins.push(
+      new CopyWebpackPlugin([
+        {
+          from: path.resolve(__dirname, 'node_modules/@webcomponents/webcomponentsjs/**/*.js'),
```

Comment:
I'm curious why is this needed?

Reply:
The webcomponents-loader loads these files at runtime, so they need to be copied over to the build output.

Reply:
ah year right